R/addRank.R

```diff
@@ -0,0 +1,17 @@
+setGeneric("addRank", signature = c("x"),
+           function(x,
+                    method = c("average", "first", "last", "random", "max", "min"))
+             standardGeneric("addRank"))
+
+setMethod("addRank", signature = c(x = "SummarizedExperiment"),
+          function(x,
+                   method = c("average", "first", "last", "random", "max", "min")){
+          col_data <- colData(x)
+          if (!is.vector(col_data)){
+            col_data <- as.vector(col_data)
+          }
+
+          ranked <- rank(col_data, na.last = TRUE , method)
```

Comment:
Calling `rank` I think it would be sufficient to provide input arguments as "..."  (`na.last` and `ties.method` can be included that way and not explicitly provided in the function argument by their name). See for instance [getPrevalence.R](https://github.com/microbiome/mia/blob/master/R/getPrevalence.R).

Keep same argument names (ties.method should remain ties.method, not switch to just method).

User needs to have access to changing na.last as well. Giving the "..." as function arguments will do the trick.